tests/setup.ts

```diff
@@ -0,0 +1 @@
+import '@testing-library/jest-dom/vitest'
```

Comment:
I use the file name `vitest-setup.js` elsewhere. Maybe `vitest-setup.ts` here?

https://github.com/jotaijs/jotai-zustand/blob/main/tests/vitest-setup.js

Reply:
Oh. I was about to change that, but look like PR is merged. :)

Reply:
I wasn't sure if you were going to change. Feel free to send new PRs.